pkg/phlaredb/block_querier.go

```diff
@@ -386,6 +386,7 @@ type Querier interface {
 	MergeByStacktraces(ctx context.Context, rows iter.Iterator[Profile]) (*phlaremodel.Tree, error)
 	MergeByLabels(ctx context.Context, rows iter.Iterator[Profile], by ...string) ([]*typesv1.Series, error)
 	MergePprof(ctx context.Context, rows iter.Iterator[Profile]) (*profile.Profile, error)
+	Series(ctx context.Context, params *ingestv1.SeriesRequest) ([]*typesv1.Labels, error)
```

Comment:
The method is supposed to be used here: https://github.com/grafana/pyroscope/blob/sg-labels/pkg/phlaredb/phlaredb.go#L333 (using the `Series` function defined below). Otherwise, local blocks on disk will not be queried

Reply:
Good catch 🤦

Fixed in https://github.com/grafana/pyroscope/pull/2431/commits/a9182acbc7d3e33580eed95c36fce2d8940c78b5.